localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_vrykul_2;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_3;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_4;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_desc;Having descended from the titans' creations themselves, the vrykul have always shown mastery in physical combat and are renowned for having been the first to domesticate proto-drakes as mounts, succeeding their iron brethren. Vrykul warriors are truly a fearsome sight whether they be upon a proto-drake or charging land-bound with a battle axe in hand.;;Da die Vrykul von den Sch�pfungen der Titanen selbst abstammen, haben sie stets ihre Meisterschaft im k�rperlichen Kampf unter Beweis gestellt und sind daf�r bekannt, dass sie die ersten waren, die Protodrachen als Reittiere domestiziert haben, als Nachfolger ihrer eisernen Br�der. Vrykul-Krieger sind ein wahrhaft furchteinfl��ender Anblick, ob sie nun auf einem Protodrachen sitzen oder mit einer Streitaxt in der Hand am Boden k�mpfen.;;;;;;;;;;;x
+ca_culture_undead_1;Unearthed Catacombs;;Exhumierte Katakomben;;;;;;;;;;;x
+ca_culture_undead_2;Unearthed Catacombs;;Exhumierte Katakomben;;;;;;;;;;;x
+ca_culture_undead_3;Unearthed Catacombs;;Exhumierte Katakomben;;;;;;;;;;;x
+ca_culture_undead_4;Unearthed Catacombs;;Exhumierte Katakomben;;;;;;;;;;;x
+ca_culture_undead_desc;Trapped between life and death and fueled by necromantic magic, the undead of this region are normally mindless, yet still powerful in numbers. Luckily, there is never a shortage of bodies to go around when it comes to scrounging around for more undead soldiers to fill a levy's ranks.;;Gefangen zwischen Leben und Tod und angetrieben von nekromantischer Magie sind die Untoten dieser Region normalerweise ohne Verstand, aber dennoch m�chtig ob ihrer schieren Zahl. Gl�cklicherweise gibt es nie einen Mangel an Leichen, wenn es darum geht, weitere untote Soldaten zu finden, um die Reihen eines Heeres zu f�llen.;;;;;;;;;;;x
+ca_culture_tuskarr_1;Tuskarr Turtle Corral;;Tuskarr-Schildkr�tengehege;;;;;;;;;;;x
+ca_culture_tuskarr_2;Tuskarr Turtle Corral;;Tuskarr-Schildkr�tengehege;;;;;;;;;;;x
+ca_culture_tuskarr_3;Tuskarr Turtle Corral;;Tuskarr-Schildkr�tengehege;;;;;;;;;;;x
+ca_culture_tuskarr_4;Tuskarr Turtle Corral;;Tuskarr-Schildkr�tengehege;;;;;;;;;;;x
+ca_culture_tuskarr_desc;The tuskarr are a strong a hardy people, descended from the frozen wastelands of Azeroth. It is only fair that their chosen mounts of war are as hardy, if not even more so, as them. Giant turtles, though slow-moving, prove themselves to be powerful beasts in combat.;;Die Tuskarr sind ein starkes und z�hes Volk, das aus den eisigen Ein�den Azeroths stammt. Es ist nur recht und billig, dass die von ihnen ausgesuchten Kriegsreittiere genauso widerstandsf�hig, wenn nicht sogar noch widerstandsf�higer sind als sie. Riesenschildkr�ten bewegen sich zwar langsam, erweisen sich aber im Kampf als m�chtige Bestien.;;;;;;;;;;;x
```

Comment:
Edited the first sentence, changing `a hardy` to `and hardy` as well as `descending` to `inhabiting`. You can't exactly claim that a wasteland is part of your family tree, hence the latter change.
```suggestion
ca_culture_tuskarr_desc;The tuskarr are a strong and hardy people, inhabiting the frozen wastelands of Azeroth. It is only fair that their chosen mounts of war are as hardy, if not even more so, as them. Giant turtles, though slow-moving, prove themselves to be powerful beasts in combat.;;Die Tuskarr sind ein starkes und z�hes Volk, das aus den eisigen Ein�den Azeroths stammt. Es ist nur recht und billig, dass die von ihnen ausgesuchten Kriegsreittiere genauso widerstandsf�hig, wenn nicht sogar noch widerstandsf�higer sind als sie. Riesenschildkr�ten bewegen sich zwar langsam, erweisen sich aber im Kampf als m�chtige Bestien.;;;;;;;;;;;x
```

Reply:
good point, done